scripts/cbsa/parse-metro-areas.ts

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs';
+import path from 'path';
+import { groupBy, values, words, deburr, sum } from 'lodash';
+import regions from '../../src/common/regions';
+const promisify = require('util').promisify;
+const csvParse = promisify(require('csv-parse'));
+
+/**
+ * This is a one-off script to parse and format metro areas from the
+ * mar2020.csv file.
+ */
+
+enum StatisticalAreaType {
+  Metro = 'Metropolitan Statistical Area',
+  Micro = 'Micropolitan Statistical Area',
+}
+
+interface CsvRow {
+  cbsaCode: string;
+  metropolitanDivisionCode: string;
+  csaCode: string;
+  cbsaTitle: string;
+  statisticalAreaType: StatisticalAreaType;
+  metropolitanDivisionTitle: string;
+  csaTitle: string;
+  countyEquivalent: string;
+  stateName: string;
+  fipsStateCode: string;
+  fipsCountyCode: string;
+  centralOutlyingCounty: string;
+}
+
+const outputPath = path.join(__dirname, '../../src/common/data/msa-data.json');
+
+async function main() {
+  const csvText = fs.readFileSync(path.join(__dirname, 'mar2020.csv'));
+  const [, ...rows] = await csvParse(csvText);
+  const metroAreas: CsvRow[] = rows
+    .map(parseRow)
+    .filter(
+      (cbsaRow: CsvRow) =>
+        cbsaRow.statisticalAreaType === StatisticalAreaType.Metro,
+    );
+
+  const groupedByCbsaCode = splitBy(metroAreas, ({ cbsaCode }) => cbsaCode);
+  const msaItems = groupedByCbsaCode.map(formatGroup);
+
+  fs.writeFileSync(
+    outputPath,
+    JSON.stringify({ metro_areas: msaItems }, null, 2),
+  );
+}
+
+function splitBy(items: CsvRow[], keyFn: (item: CsvRow) => string) {
+  return values(groupBy(items, keyFn));
+}
+
+function formatUrl(name: string) {
+  return name
+    .toLowerCase()
+    .split(', ')
+    .map(part => deburr(words(part).join('-')))
+    .join('_');
+}
+
+function formatGroup(groupItems: CsvRow[]) {
```

Comment:
It might be nice to define an interface for the return object just to make it explicit what fields we expect to be in the json file

Reply:
I'm going to update once we merge your reorganization!